packages/cra-template/template/src/modules/application/components/App.jsx

```diff
@@ -40,24 +19,15 @@ const routes = [
         component: Login,
     },
     // @use-auth-module-end
-    {
-        component: NoMatchPage,
-    },
```

Comment:
what about replacing this with

```jsx
{
   component: <Redirect to="/" />
}
```

I'd also move routes definition to config as it is common practice on projects

```js
routes: {
   home: '/'
}

...

path: config.routes.home
```

Reply:
I agree, let's define them in the global config